.github/workflows/generate_dolphin_weights_files.yml

```diff
@@ -0,0 +1,283 @@
+    runs-on: ${{ needs.start-node-builder-current.outputs.runner-label }}
+    strategy:
+      matrix:
+        benchmark:
+          -
+            extrinsic:
+              id: '*'
+              name: frame_system
+            pallet:
+              id: frame_system
+              name: frame_system
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_balances
+            pallet:
+              id: pallet_balances
+              name: pallet_balances
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_collective
+            pallet:
+              id: pallet_collective
+              name: pallet_collective
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_democracy
+            pallet:
+              id: pallet_democracy
+              name: pallet_democracy
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_membership
+            pallet:
+              id: pallet_membership
+              name: pallet_membership
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_multisig
+            pallet:
+              id: pallet_multisig
+              name: pallet_multisig
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_scheduler
+            pallet:
+              id: pallet_scheduler
+              name: pallet_scheduler
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_session
+            pallet:
+              id: pallet_session
+              name: pallet_session
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_tx_pause
+            pallet:
+              id: pallet_tx_pause
+              name: pallet_tx_pause
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_utility
+            pallet:
+              id: pallet_utility
+              name: pallet_utility
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_timestamp
+            pallet:
+              id: pallet_timestamp
+              name: pallet_timestamp
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: pallet_treasury
+            pallet:
+              id: pallet_treasury
+              name: pallet_treasury
+            iterations: 20
+          -
+            extrinsic:
+              id: '*'
+              name: manta_collator_selection
+            pallet:
+              id: manta_collator_selection
+              name: manta_collator_selection
+            iterations: 20
```

Comment:
Could you check if we can make the matrix dynamic and pass a list of the pallets you want to benchmark as input.
If there's no input we can have this as default. Kind of out of scope, but we'd be able to dedupe the 3 workflows if we can do that.